docs/integrations/azure-devops-pipeline.md

```diff
@@ -9,10 +9,12 @@ Scan has a best-in-class integration for Azure Pipelines with our dedicated [ext
 ```yaml
 - script: |
     docker run \
+      -e "VDB_HOME=/tmp" \
```

Comment:
Perhaps a different path such as `$(Build.ArtifactStagingDirectory)/vdb` instead of `/tmp`?

Reply:
Outputs:

[VDB_HOME=$(Build.ArtifactStagingDirectory)/vdb](https://github.com/ShiftLeftSecurity/scan-docs/files/6052757/output_with_Build.ArtifactStagingDirectory.txt)

[VDB_HOME=/tmp](https://github.com/ShiftLeftSecurity/scan-docs/files/6052758/output_with_tmp.txt)

Reply:
Since `$(Build.ArtifactStagingDirectory)` is mounted as `/reports` within the container this variable should be specified as `/reports/vdb`. All the directories passed to scan are based on mounted volumes since it cannot access the host directory structures.

Reply:
OK. It worked.